src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -219,6 +219,12 @@ public function buildView(FormView $view, FormInterface $form, array $options)
             // POST request.
             $view->vars['full_name'] .= '[]';
         }
+
+        // Provide a default message when choices var is empty
+        if (empty($view->vars['choices']) && $options['expanded']) {
+            $view->vars['empty_choices'] = null !== $options['empty_choices']
+                ? $options['empty_choices'] : '<em>No choice available</em>';
```

Comment:
:+1:

I looked at the other components and only Security Core has its own translation domain `security`. For now form component has a validator catalogue with id's taking into account the ones of validator component.

Anyway I don't think that this message belongs to validator domain but the point is I wonder if I add a  `form.lang.xliff`, isn't this going to mess with apps when users have their own `form` domain ? Should't it be prefixed like 'sf_from.lang.xliff` ?

@xabbuh @aitboudad can you please light me on this ?